src/components/ha-base-time-input.ts

```diff
@@ -77,27 +79,27 @@ export class HaBaseTimeInput extends LitElement {
   /**
    * Label for the day input
    */
-  @property() dayLabel = "";
+  @property() dayLabel = "Day";
```

Comment:
These should be translatable.

Reply:
Is this the way to do it?
`this.hass!.localize("ui.components.<something?>")`

Reply:
Yeah, the translations should be added to https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json
I would suggest ui.components.selectors.time.<day/...>. After that you can use them in the code that way. 

Please note that this.hass may be uninitialized at the time you're assigning dayLabel and so on. You may need to remove the default value so if not defined it ends up as being undefined and then check within the render if the dayLabel is defined and if not, use this.hass.localize("label") to fill in the default. 😃

Reply:
I've taken the easy way out and made the labels empty again.
I declare it out of scope 😄

passing around hass objects that may or may not exist is a little bit too much for this merge request.

Reply:
Ok, it shouldn't be much passing around, I'll take a look if I can add it.

Reply:
Do you want to add it to this pull request or to a fresh one?

Reply:
I just had a look and it's a component that doesn't have access to hass. So, let's declare it out of scope. 
They should be passed in with the properties by the components that do have access to hass.